### ISSUE TITLE
Milicores consumed by container from cadvisor.

### ISSUE THREAD

**Issue by SamratAmate** [google/cadvisor#2633]
@dashpole  How do I get milicores used by containers inside docker stack at particular time from cadvisor?  Which metrics query I should be using? My query is exactly the data that we see on cadvisor webUI. 
![Capture](https://user-images.githubusercontent.com/40057697/88506900-b1337500-cff8-11ea-94ae-fc62df6fd301.JPG)

Note: cadvisor is running as a separate container outside of stack.

**Comment by dashpole** [comment]
I'm not sure what "inside stack" vs "outside stack" means.  cAdvisor must be running on the machine with the other containers it is monitoring.

Are you looking for a different way to visualize the metrics other than the UI?  I'm not quite sure what you are asking

**Comment by SamratAmate** [comment]
@dashpole  Sorry I wasn't very clear at first. Let me try again.

Yes, cadvisor is running on same machine on which I have other containers whose performance I want to monitor.
I am using prometheus to collect the data from cadvisor.

Now, what I want is to draw the charts same as what we see on the docker webUI (as shown in above images). 
So, which metrics query I should use, on Prometheus, to get the milicores used by any container at specific time?

Do we have anything like below which will give me data I want:
container_cpu_**milicores_used**?

**Comment by dashpole** [comment]
I believe the metric you want is container_cpu_usage_seconds_total, which is measured in core-seconds.

**Comment by SamratAmate** [comment]
So the value we get from container cpu_usage_seconds_total  can also be considered as number of milicores consumed?

**Comment by dashpole** [comment]
It is a cumulative metric, so you have to take a rate over two samples.  That would give you cpu usage in cores.  You can multiply by 1000 to get millicores if you'd like.

**Comment by dims** [comment]
Closing this issue as it has been open since before January 1, 2024. If this is still relevant, please feel free to reopen or create a new issue.
